docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,83 @@
+---
+order: 5
+title: Emergency Upgrades
+---
+
+## **Introduction**
+
+This article goes into the general flow of events for an emergency upgrade, pre-upgrade and during the upgrade for all validators.
+
+Emergency upgrades are an unfortunate necessity for all validators when running a PoS node.
+
+When a vunerability is discovered and responsibily disclosed, the vunerability will be reported to one of the channels described in the [Security.md](../../SECURITY.md).
+
+Once the vulnerability is patched and tested, the core team will reach out to all validators to update their nodes with the latest version.
+
+### **Cosmos Hub Upgrade Information**
+
+Depending on the severity of the vulnerability, the core teams and validators can take different actions.
+
+The core teams will try reach out **48 hours** or more before a scheduled upgrade, so that validators have time to inspect the changes and also adjust their internal planning accordingly.
+
+If the change is **non** state-breaking then the patch can be applied in a un-cordinated manner by validators asap.
```

Comment:
```suggestion
If the change is **non** state-breaking then the patch can be applied in a un-coordinated manner by validators asap.
```

Reply:
I'm still not confident in the division between state breaking and non-state-breaking changes. There could be security patches that are not state breaking but they still require a coordinated upgrade, for example because of the risk of loss of funds.

My preferred distinction is: requires a coordinated upgrade or a coordinated upgrade is not required.

Reply:
Went with Greg's suggestion.